tasks/add-missing-file-checksums/package.json

```diff
@@ -40,7 +40,7 @@
     }
   },
   "dependencies": {
-    "@cumulus/aws-client": "2.0.1",
+    "@cumulus/aws-client": "2.0.2",
     "@cumulus/cumulus-message-adapter-js": "1.2.0"
```

Comment:
Why is this still `1.2.0` all over the place? 🤔 If `1.3.1` is just YAGNI in these places, then that's fine.

Reply:
Do you mean 2.0.1?

Reply:
I'm annoyed by that one because I thought I searched for this.    *looks*

Reply:
@nemreid  because https://github.com/nasa/cumulus/pull/1802 was merged after 2.0.1. 

All the 1.3.x CMA-js update did was pull in the typescript version of cma-js. 

https://github.com/nasa/cumulus-message-adapter-js/blob/master/CHANGELOG.md


That doesn't feel like a critical/backportable fix, and cma-js 1.2/1.3 should be interop.

Reply:
Great, thanks for answering that.